content/@i18n/ja/concepts/networks-and-servers/parallel-networks.md

```diff
@@ -33,7 +33,7 @@ Ripple社は、TestnetとDevnetでメインサーバーを運用しています
 ## 関連項目
 
 - **ツール:**
-  - [XRP Testnet Faucet](xrp-test-net-faucet.html)
+  - [XRP Testnet Faucet](/dev-tools/xrp-faucets)
```

Comment:
I'd like to handle link migration separately because we're going to have to do mass replacements of other links and I want to minimize the occasions for merge conflicts. How hard would it be to change the PR to exclude all these changes to update links in `.md` files?

Reply:
Very doable :)

Reply:
Done :)